ros/src/computing/perception/detection/lidar_detector/packages/lidar_euclidean_cluster_detect/CMakeLists.txt

```diff
@@ -65,22 +65,8 @@ add_executable(lidar_euclidean_cluster_detect
 
 find_package(CUDA)
 find_package(Eigen3 QUIET)
-####### Melodic Checks
```

Comment:
You could make a couple of macros for this (one for just the CMake check and one that has both CMake and Eigen checks) and put them in the `autoware_build_flags-extras.cmake` file.